Example/ViewController.swift

```diff
@@ -83,11 +83,14 @@ final class ViewController: UIViewController {
     }
     
     @IBAction func metadata(_ sender: Any) {
-        let newMetadata = ModernAVPlayerMetadata(title: "Updated title",
+        let formatter = DateFormatter()
+        formatter.dateFormat = "HH:mm:ss"
+        let timeStamp = formatter.string(from: Date())
+        let newMetadata = ModernAVPlayerMediaMetadata(title: "Updated title (\(timeStamp))",
```

Comment:
hard to easily read seconds so I suggest to replace all title string by the timeStamp